AdminWebsite/AdminWebsite/ClientApp/src/app/services/search.service.spec.ts

```diff
@@ -94,20 +95,34 @@ participant2.first_name = 'participant2FirstName';
 participant2.last_name = 'participant2LastName';
 participant2.display_name = 'participant2DisplayName';
 participant2.email = 'participant2Email';
-
 const participantList: ParticipantModel[] = [participant1, participant2];
 
+const staffMember1 = new ParticipantModel();
```

Comment:
a suggestion here, upto you if you want to change it but alternatively I think you could do this: 
`
const staffMember1:  ParticipantModel = {
first_name : 'StaffMember1FirstName',
last_name : 'StaffMember1LastName',
display_name : 'StaffMember1DisplayName',
email : 'StaffMember1Email'
};
`
But this is down to personal preference and I dont know if there is any performance gain anyways